### ISSUE TITLE
documentation on dependency injection provides a non working example

### ISSUE THREAD

**Issue by silkentrance** [pana-cc/mocha-typescript#102]
@Eronana can you provide us with a fix for this?

In the provided example it reads (README.md)

```
import { assert } from "chai";
import { Service } from "typedi";
import "../../di/typedi";
import { register, suite, test } from "../../index";
```

Here, the relative paths must be replaced by absolute package paths, e.g.

```
import "mocha-typescript/di/typedi";
import { register, suite, test } from "mocha-typescript";
```

The documentation also lacks the information that the user needs to add typedi to his or her dev/dependencies.

**Comment by silkentrance** [comment]
@Eronana ping

**Comment by Eronana** [comment]
sorry for late response.
so you want me fix `test/it/fixtures/di.typedi.suite.ts` and copy fixed test code to document, right?

**Comment by silkentrance** [comment]
@Eronana the test is ok, the example in the readme is wrong.

**Comment by silkentrance** [comment]
Fixed with #115.
